### PR TITLE
implements new material tooltip

### PR DESCRIPTION
**Pull request by bdbch**
This PR implements the new `@material-ui/core` tooltips for all simple tooltips which don't need special markup or styling and removes several data-tip and ReactTooltip uses.

### CODE REVIEW

File: src/components/Table/TableHeaderColumn.jsx [hunk 1]
```diff
@@ -27,17 +24,15 @@ const TableHeaderColumn = ({
...
           style={{ color: column.color, width: '100%', textAlign: getColStyle(column).textAlign }}
         >
-          {column.displayName}
-          {column.sortFn && getSortIcon(sortState, sortField, column.field, { height: 14, width: 14 })}
-          {column.tooltip &&
-          <ReactTooltip id={tooltipId} place="top" type="light" effect="solid">
-            {column.tooltip}
-          </ReactTooltip>
-          }
+          { !column.tooltip ? column.displayName : (
```
Comment (howardchung) [inline src/components/Table/TableHeaderColumn.jsx]: Do we need the sort icon here?

**Comment by howardchung** [comment]
Looks good, one question that I commented on

File: src/components/Table/TableHeaderColumn.jsx (hunk 1 above)
Comment (bdbch) [reply src/components/Table/TableHeaderColumn.jsx]: True! will re-add it

**Comment by bdbch** [comment]
could be reviewed again @howardchung

**Pull request merged**